src/toil/jobStores/googleJobStore.py

```diff
@@ -199,7 +199,9 @@ def load(self, jobStoreID):
             jobString = self._readContents(jobStoreID)
         except NoSuchFileException:
             raise NoSuchJobException(jobStoreID)
-        return pickle.loads(jobString)  # UPDATE bz2.decompress(
+        job = pickle.loads(jobString) 
+        job.assignConfig(self.config)
```

Comment:
Could a comment be added as to what `job.assignConfig` is doing here?

Reply:
Sure.